src/lexers/javascript-lexer.js

```diff
@@ -17,10 +19,9 @@ export default class JavascriptLexer extends BaseLexer {
     walk.simple(
       acorn.parse(content, this.acornOptions),
       {
-        Expression(node) {
+        CallExpression(node) {
```

Comment:
Two reasons:
1. We were already checking if it were a CallExpression, and not doing anything on any other types.
2. Setting it to CallExpression lets it work in both js and jsx syntax.